README.md

```diff
@@ -172,6 +171,24 @@ TC53][tc53], do not have an origin on which to build a same-origin-policy, and
 have elected to build their security model on isolated evaluators, through the
 high-level Compartment interface.
 
+### Isolating unreliable code
+
+The way modern software is composed has already undermined the validity of the assumption that every author participating has their intentions well aligned for the benefit of the software working correctly. A whole new level of unreliability is now added with the popularity of _coding agents_ and _vibe coding_ where creating syntactically valid but effectively unpredictable JavaScript and integrating it into existing software to check whether it seems to implement the desired functionality is becoming a popular way of building software.
+
+It is not an entirely new concern, as test runners have been concerned with isolating test cases to avoid them relying on global side-effects of other test cases. It is now a concern for a much wider audience with more at stake.
+
+With `Global` constructor comes the ability to isolate fragments of the application in a way that unreliable code cannot rely on shared global state without the maintainer of the software knowing about it.
+AI generated sources from independently working agents can come with colliding names for global variables to use and may need separate global scopes to collaborate or coexist. Similarly a misguided attempt at an inline polyfill by an AI or a package author could be prevented by freezing the contents of a new global in which the unreliable code subsequently runs.
+Using a new global instead of a new Realm avoids the issues like identity discontinuity impeding the composition of software where function calls need to happen across the isolated and non-isolated code.
+
+### Incremental or in-context execution
+
+There are tools that currently use much more complex and costly mechanisms (similar to the ones described in [Domain Specific Languages](#domain-specific-languages) among other) to provide the ability to execute fragments of JavaScript code in a very specific context of the tool.
+
+That includes REPLs, inline code execution results in editors (eg. [Quokka.js](https://quokkajs.com/)) and various use cases of IDEs in the browser.
+
+Maintaining the global state between executions of user-provided code snippets would benefit from a `Global` constructor.
```

Comment:
Consider including a usage example that shows preservation of a global contour. You might want to flesh out the API for that since the global contour shouldn’t be preserved between calls to `eval` ordinarily. Maybe it needs to be an option. The language has no other mention of global contour yet.

Reply:
I was certain that the REPL usecase is limited to creating a single `global` distinct from the global in which the REPL implementation (IO handling and other instrumentation)  is running and calling `thatGlobal.eval()` on each line of input therefore collecting the contour within `thatGlobal` while not polluting the global of the REPL implementation (and the other way around).
The example seems trivial and not requiring much more design. What am I missing?